test/gululog_idx_SUITE.erl

```diff
@@ -20,6 +20,12 @@
 
 -define(config(KEY), proplists:get_value(KEY, Config)).
 
+-record(idx, { version :: logvsn()
```

Comment:
idx is an 'opaque' you don't want to expose outside of gululog_idx.erl, not even for testing.
if it's done in UT, it's fine. but not in CT.
do black-box testing here, use only the API functions to verify delete result.